apps/evm/src/clients/api/queries/getIsolatedPools/index.ts

```diff
@@ -83,31 +75,29 @@ const getIsolatedPools = async ({
   }
 
   // Extract token records and addresses
-  const [vTokenAddresses, underlyingTokens, underlyingTokenAddresses] = poolResults.value.reduce<
-    [string[], Token[], string[]]
-  >(
+  const [vTokenAddresses, underlyingTokens] = isolatedPoolsData.pools.reduce<[string[], Token[]]>(
     (acc, poolResult) => {
       const newVTokenAddresses: string[] = [];
       const newUnderlyingTokens: Token[] = [];
       const newUnderlyingTokenAddresses: string[] = [];
 
-      poolResult.vTokens.forEach(vTokenMetaData => {
+      poolResult.markets.forEach(market => {
         const underlyingToken = findTokenByAddress({
           address:
             // If underlying asset address is the null address, this means the VToken has no
             // underlying token because it is a native token
-            areAddressesEqual(vTokenMetaData.underlyingAssetAddress, NULL_ADDRESS)
+            areAddressesEqual(market.underlyingTokenAddress, NULL_ADDRESS)
               ? NATIVE_TOKEN_ADDRESS
-              : vTokenMetaData.underlyingAssetAddress,
+              : market.underlyingTokenAddress,
```

Comment:
This logic isn't needed anymore since it's present here already: https://github.com/VenusProtocol/venus-protocol-interface/pull/3156/files#diff-c9a8841d55105945315df70e596199b85a70aa24bedaf507f506ae8b69a6217cR20